hkpy/hkbase/hkrepository.py

```diff
@@ -66,17 +66,21 @@ def create_transaction(self, id_: Optional[str]=None) -> HKTransaction:
 
         return HKTransaction(id_=generate_id(self))
 
-    def add_entities(self, entities: Union[HKEntity, List[HKEntity]], transaction: Optional[HKTransaction]=None) -> None:
+    def add_entities(self, entities: Union[HKEntity, List[HKEntity]], transaction: Optional[HKTransaction] = None, force_add: Optional[bool] = False) -> None:
         """ Add entities to repository.
-        
+
         Parameters
         ----------
         entities : (Union[HKEntity, List[HKEntity]]) entity or list of entities
         transaction : (Optional[HKTransaction]) connection transaction
+        force_add: (Optional[bool] flag to bypass verification of preexisting entities and add all assuming they are new
         """
 
         url = f'{self.base._repository_uri}/{self.name}/entity/'
 
+        if force_add:
+            url = f'{url}?forceAdd=true'
+
```

Comment:
I think is better to use requests' [prescribed way to deal with query params](https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls)